lap3dkernel/c++SIMD/lap3dkernel.cpp

```diff
@@ -155,11 +234,26 @@ int main (void)
   Real t = omp_get_wtime()-ts;
   Real ans = 0;
   for(auto& x : pot) ans+=x;    // this answer checks it agrees btw methods, to 15 digits
-  std::cout << std::setprecision(15);
+  std::cout << std::setprecision(16);
```

Comment:
the result differs in 15th digit with rounding. If print to 16 digits, 15th digit match, 16th digit does not match.